vars/abapEnvironmentPipelineStageIntegrationTests.groovy

```diff
@@ -31,29 +32,42 @@ void call(Map parameters = [:]) {
         .mixinGeneralConfig(script.commonPipelineEnvironment, GENERAL_CONFIG_KEYS)
         .mixinStageConfig(script.commonPipelineEnvironment, stageName, STEP_CONFIG_KEYS)
         .mixin(parameters, PARAMETER_KEYS)
-        .addIfEmpty('confirmDeletion', true)
+        .addIfEmpty('confirmDeletion', false)
         .addIfEmpty('debug', false)
         .addIfEmpty('testBuild', false)
+        .addIfEmpty('integrationTestOption', 'addOnDeployment')
         .use()
 
     if (config.testBuild) {
         echo "Stage 'Integration Tests' skipped as parameter 'testBuild' is active"
     } else {
```

Comment:
if (config.testBuild) {
        echo "Stage 'Integration Tests' skipped as parameter 'testBuild' is active"
        return null
    } 
Isn't return null possible so we can avoid the complex else branch?